tmt/cli.py

```diff
@@ -291,6 +291,12 @@ def main(
          """)
 @environment_options
 @verbosity_options
+@option(
+    '--debug-host',
+    default=False,
+    is_flag=True,
+    help='If set, tmt will log facts about the machine on which it was running.'
+    )
```

Comment:
Is a special option for this needed? Couldn't we provide the info by default? It's not that much extra data in the debug log, right?

Reply:
It's not needed. In 199351da, I dropped the option, and changed the logging to use `debug` instead of `verbose`. Now it's behind `-ddd`.